tests/Unit/Domain/Creators/Test_Rectangle.cpp

```diff
@@ -142,53 +175,117 @@ void test_rectangle() {
            {Direction<2>::lower_eta(), {0, aligned_orientation}},
            {Direction<2>::upper_eta(), {0, aligned_orientation}}}},
       std::vector<std::unordered_set<Direction<2>>>{{}});
+
+  const creators::Rectangle periodic_rectangle_boundary_conditions{
```

Comment:
(optional) I think these tests would benefit from a few added scopes like `{ INFO("Periodic"); ...}`, so they print a little context when they fail and so variable names don't have to be prefixed with e.g. `periodic`. I'm happy to merge this as-is, and suggest to add some more scopes in the upcoming domain creator PRs.